README.md

```diff
@@ -10,6 +10,7 @@ python3 -m venv .venv
 source .venv/bin/activate
 pip3 install --upgrade pip
 pip3 install -r requirements.txt
+python -m pip install -U matplotlib
```

Comment:
nope. the process for creating the `requirements.txt` file is:

- create a fresh virtual environment
- run all scripts and install **only** the required dependencies
- Once you are content that you have all of your dependencies installed, then run `pip3 freeze > requirements.txt`